harness/determined/cli/task.py

```diff
@@ -217,6 +301,105 @@ def logs(args: Namespace) -> None:
                     *common_log_options,
                 ],
             ),
+            Cmd(
+                "create",
+                create,
+                "create task (EXPERIMENTAL: This command should only be used in dev environments)",
```

Comment:
If you are not intending on this CLI endpoint being released to users, use `argparse.SUPPRESS` as the help text, so that it doesn't show up in the `--help` output at all.